frames/src/main/java/com/checkout/frames/utils/constants/InputFieldConstants.kt

```diff
@@ -1,5 +1,5 @@
 package com.checkout.frames.utils.constants
 
 internal object InputFieldConstants {
-    internal const val EXPIRY_DATE_MAX_LENGTH = 4
+    internal const val EXPIRY_DATE_MAXIMUM_LENGTH = 4
 }
```

Comment:
Probably, it's more related to expiry date component constants than to the base input field component.